run.py

```diff
@@ -1,8 +1,9 @@
 from sanic import Sanic
 from app import create_app
 
+from config import config
 
-app = create_app()
+app = create_app("development")
```

Comment:
Why this magic string here? "development". what does it represent?

Reply:
the app is created with a default config value. Possible values are : development, testing and production. When the app will go into production this will change.